test/jdk/java/awt/geom/Path2D/GetBounds2DPrecisionTest.java

```diff
@@ -189,7 +189,7 @@ private static String toUniformString(double value) {
         int DIGIT_COUNT = 40;
         String str = decimal.toPlainString();
         if (str.length() >= DIGIT_COUNT) {
-            str = str.substring(0,DIGIT_COUNT-1)+"…";
+            str = str.substring(0,DIGIT_COUNT-1)+"...";
         }
```

Comment:
How did you test this ? Please say more than tiers 1-3 .. because this test isn't run until tier4.

Reply:
I did not test tier4. Will do so now. Thanks!